util/src/main/java/tc/oc/pgm/util/material/Materials.java

```diff
@@ -32,6 +32,7 @@ public interface Materials {
   Material BOOK_AND_QUILL = parse("BOOK_AND_QUILL", "WRITABLE_BOOK");
   Material EYE_OF_ENDER = parse("EYE_OF_ENDER", "ENDER_EYE");
   Material FIREWORK = parse("FIREWORK", "FIREWORK_ROCKET");
+  Material BANNER = parse("BANNER");
```

Comment:
```suggestion
  Material BANNER = parse("BANNER", "LEGACY_BANNER");
```
for modern support i believe you need this